File: smiex_api/smiex_api/Models/Freight.cs
using System;
using System.Collections.Generic;

namespace smiex_api.Models;

public partial class Freight
{
    public int IdFreight { get; set; }

    public string? Tipus { get; set; }

    public virtual ICollection<Oferte> Ofertes { get; set; } = new List<Oferte>();
}

File: simex_app/app/src/main/java/com/example/simex_app/data/models/Freight.kt
package com.example.simex_app.data.models

data class Freight(
    val idFreight: Int,
    val tipus: String?,

    // Relación con las Ofertas (Uno a Muchos)
    // Usamos emptyList() para que la App no se rompa si no hay ofertas asociadas
    val ofertes: List<Oferte> = emptyList()
)